.dotnet/tests/Batch/BatchTests.cs

```diff
@@ -22,14 +23,14 @@ public BatchTests(bool isAsync) : base(isAsync)
     }
 
     [Test]
-    public async Task ListBatchesProtocol()
+    public void ListBatchesProtocol()
```

Comment:
Would it make sense to provide tests for both sync and async for this?  IIRC I ended up writing separate tests for these in the SCM test project because there wasn't a clean way to handle IEnumerable and IAsyncEnumerable tests in the same methods.  You can see examples of those [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/System.ClientModel/tests/Convenience/PageCollectionTests.cs) and [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/System.ClientModel/tests/Convenience/PageCollectionScenarioTests.cs).